### ISSUE TITLE
Package org.hsqldb.error does not exist

### ISSUE THREAD

**Issue by attacomsian** [marytts/marytts-testing#5]
I'm trying to build the **[marytts-testing](https://github.com/attacomsian/marytts-testing)** at **falken-4** but it is failing with the following error.

```
:bleeding-marytts:marytts-signalproc:classes UP-TO-DATE
:bleeding-marytts:marytts-signalproc:jar UP-TO-DATE
:bleeding-marytts:marytts-runtime:compileJava
/local/attashah/marytts-testing/bleeding-marytts/marytts-runtime/src/main/java/marytts/modules/phonemiser/TrainedLTS.java:29: error: package org.hsqldb.error does not exist
import org.hsqldb.error.Error;
                       ^
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
1 error
:bleeding-marytts:marytts-runtime:compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':bleeding-marytts:marytts-runtime:compileJava'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 9 mins 42.302 secs
```

I tried to upgrade Gradle wrapper to`2.14` from `2.7` but problem still persists. After googling, I found that it my be related to [shell script](http://stackoverflow.com/questions/13970119/hsqldb-alias-doesnt-exist-linux-sh-script) but not sure. 

@seblemaguer any possible hack or suggestion to fix this?  Thanks.

**Comment by seblemaguer** [comment]
normally last commit should solve this. Can you try to update your fork based on that ?

**Comment by psibre** [comment]
fixed in d82f9156298eca400712f54819c4705ea4009eb6
